modules/ROOT/pages/_partials/commons/common-p2psync-websocket.adoc

```diff
@@ -18,102 +18,104 @@ endif::[]
+. Replicator and Listener engage in the configured security protocol exchanges to confirm connection
 . If connection is confirmed then replication will commence, synchronizing the two data stores.
 
 ====
 
-As you can see this involves configuring a {p2psync-websocket-using-passive--xref} and an
-*** {p2psync-websocket-using-active--xref}; you can find the instructions to do that on those two links and see a simple listener configuration in <<simple-configuration>>
+Here you can see configuration involves a {p2psync-websocket-using-passive--xref} and an {p2psync-websocket-using-active--xref} and a user-friendly Listener configuration in <<simple-configuration>>
 
-You can also learn more about how to implement peer-to-peer synchronization by referring to our tutorial -- see: {tutorial--xref--p2psync}.
+You can also learn how to implement Peer-to-Peer synchronization by referring to our tutorial -- see: {tutorial--xref--p2psync}.
 
 == Features
 
-Couchbase Lite for {param-title}'s peer-to-peer synchronization solution provides support for cross-platform synchronization; for example, between Android and iOS devices.
+Couchbase Lite for {param-title}'s Peer-to-Peer synchronization solution provides support for cross-platform synchronization, for example, between Android and iOS devices.
 
-Each listener instance serves one Couchbase Lite database.
-But there is no hard limit on the number of listener instances that you can associate with a database.
+Each listener instance serves one Couchbase Lite database. However, there is no hard limit on the number of Listener instances you can associate with a database.
 
-Nor does having a listener on a database prevent you opening replications to the other clients.
-So, for example, whilst listening for connections, a listener is able to actively initiate replications to other listeners.
-This can be for the same or a different database.
+Having a Listener on a database still allows you to open replications to the other clients. 
+For example, a Listener can actively begin replicating to other Listeners while listening for connections. 
+These replications can be for the same or a different database.
 
-The listener will automatically select a port to use or will use a user-specified port.
-It will also listen on all available networks, unless you specify a specific network to use.
+The Listener will automatically select a port to use or a user-specified port.
+It will also listen on all available networks, unless you specify a specific network.
 
 === Security
 
-peer-to-peer synchronization supports encryption and authentication over TLS with multiple modes, including:
+Couchbases Lite's Peer-to-Peer synchronization supports encryption and authentication over TLS with multiple modes, including:
 
-* No encryption, that-is, clear text.
-* CA Cert
+* No encryption, for example, clear text.
+* Certificate Authority (CA)
```

Comment:
Careful: "CA Cert" is a technical phrase.

A "CA" is the authority. 
A "CA Cert" is a specific sort of certificate relating to the authority.